core/pkg/hostsensorutils/hostsensordeploy.go

```diff
@@ -92,6 +93,31 @@ func (hsh *HostSensorHandler) Init(ctx context.Context) error {
 	return nil
 }
 
+// checkNamespaceWasPresent check if the given namespace was already present on kubernetes.
+// Return true in case it find the namespace on the list, false otherwise.
```

Comment:
[Required]: We need to make sure the status of the namespace is `Active`.

Reply:
Yes, it makes sense.